### ISSUE TITLE
Selected Tab in sideNav doesn't have sufficient contrast. 

### ISSUE THREAD

**Issue by bhanupradeep7** [vmware/clarity#6031]
## Describe the bug

Selected tab in the Side nav doesn't have sufficient contrast. This issue is present in clarity v5 as well
Currently the background-color is #fafafa and selected tab background color is #d8e3e9.

Contract checker : https://www.levelaccess.com/color-contrast-checker/
## How to reproduce

https://clarity.design/angular-components/sidenav/#code-examples

1.  Go to https://clarity.design/angular-components/sidenav/#code-examples
2.  See the example provided. 

## Expected behavior

Current Contrast ratio between selected tab background and Page background is 1.25:1 . This value has to be more than 3:1.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [x] v3.x
- [x] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
Angular 10
Clarity 5

**Device:**

- Type: MacBook
- OS: Catalina
- Browser: chrome
- Version: 90

**Comment by hippee-lee** [comment]
Thanks for reporting this issue. We will look into it further. 

@scroniser and @colinreedmiller - are you able to weigh in here on the best way to update the selected tab color to meet the 3:1 contrast ratio?

**Comment by scroniser** [comment]
This is being tracked internally. There are a couple ways to solve this that I can suggest (though I will defer to Colin & the design team): include a border which provides sufficient contrast with the background or invert the colors (light text on a dark background).

**Comment by hippee-lee** [comment]
Thanks @scroniser - since this is being tracked internally I'm going to close it here. 

Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
